Aztec/Classes/TextKit/ParagraphStyle.swift

```diff
@@ -191,6 +191,23 @@ open class ParagraphStyle: NSMutableParagraphStyle, CustomReflectable {
 
         return CGFloat(depth) * Metrics.listTextIndentation
     }
+    
+    /// The amount of indent for the nested blockquote of the paragraph if any.
+    ///
+    public var blockquoteNestedIndent: (depth: Int, indent: CGFloat) {
+        let listAndBlockquotes = properties.filter({ property in
+            return property is Blockquote || property is TextList
+        })
+        var depth = 0
+        for position in (0..<listAndBlockquotes.count).reversed() {
+            if listAndBlockquotes[position] is Blockquote {
+                depth = position
+                break
+            }
+        }
+        let indent = CGFloat(depth) * Metrics.listTextIndentation
+        return (depth, indent)
+    }
```

Comment:
Can we please split this in two methods, one to calculate  the depth and another for the indent value?

The one for indent can use the method for depth inside.

Reply:
yeah, I was just following the established pattern for these vars as they were used for the list and thought a tuple was an elegant solution to get the depth info, but I can split it up. Actually it reminds me I need to merge it with the `blockquoteIndent` var right above it

Reply:
ok interesting, when I have only `blockquoteIndent` var rather than both that and `blockquoteNestedIndent` then the background rect shifts as you can see here. I guess I will keep both vars to keep original behavior. 
<img width="584" alt="Screen Shot 2019-12-16 at 5 13 53 PM" src="https://user-images.githubusercontent.com/10858053/70956117-dbb35d80-2027-11ea-905d-94e2a91a6530.png">

Reply:
well you can see the solution I came up with; just one var, but the block quote background always starts at indent of 0